src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -187,33 +187,26 @@ public class Access extends AbstractApiBean {
     @Inject
     MakeDataCountLoggingServiceBean mdcLogService;
     
-    
-    private static final String API_KEY_HEADER = "X-Dataverse-key";    
-    
     //@EJB
     
     // TODO: 
     // versions? -- L.A. 4.0 beta 10
     @Path("datafile/bundle/{fileId}")
     @GET
     @Produces({"application/zip"})
-    public BundleDownloadInstance datafileBundle(@PathParam("fileId") String fileId, @QueryParam("fileMetadataId") Long fileMetadataId,@QueryParam("gbrecs") boolean gbrecs, @QueryParam("key") String apiToken, @Context UriInfo uriInfo, @Context HttpHeaders headers, @Context HttpServletResponse response) /*throws NotFoundException, ServiceUnavailableException, PermissionDeniedException, AuthorizationRequiredException*/ {
+    public BundleDownloadInstance datafileBundle(@PathParam("fileId") String fileId, @QueryParam("fileMetadataId") Long fileMetadataId,@QueryParam("gbrecs") boolean gbrecs, @Context UriInfo uriInfo, @Context HttpHeaders headers, @Context HttpServletResponse response) /*throws NotFoundException, ServiceUnavailableException, PermissionDeniedException, AuthorizationRequiredException*/ {
```

Comment:
Most api calls don't add the key as a QueryParam so I've removed them in this class.

Reply:
Note to self: we need to retire this "bundle access" method.